### PR TITLE
Setup and Mobile First

### PR DESCRIPTION
**Pull request by esmiz-dream**
### About Me and My Recent Work

In This Pull Request I:
- Added My Recent Work section
- Added About Me section
- Added Get in Touch section
- Added styling of the above sections

### CODE REVIEW

**Review by osamaashraf6 — COMMENTED** [review]
Hi @esmiz-dream  😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

You have submitted a project, that there is a linter error in the stylelint ,so kindly try to fix this error😊

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -48,26 +48,281 @@
...
-                    <li><a href="https://github.com/esmiz-dream" target="_blank"></a><img src="./images/Vector-2.png"
+                    <li><a href="https://linkedin.com/in/esmael-mohammed-10ba5a197" target="_blank" rel="noopener"><img
+                                src="./images/linkedin.png" alt="linkedin"></a></li>
+                    <li><a href="https://github.com/esmiz-dream" target="_blank" rel="noopener"></a><img src="./images/Vector-2.png"
                             alt="Pinter"></a></li>
-                    <li><a href="https://twitter.com/esmaelmohamme16" target="_blank"><img src="./images/twiter.png"
+                    <li><a href="https://twitter.com/esmaelmohamme16" target="_blank" rel="noopener"><img src="./images/twiter.png"
                                 alt="Twiter"></a></li>
-                    <li><a href="https://github.com/esmiz-dream" target="_blank"><img src="./images/m.png" alt="M"></a>
+                    <li><a href="https://github.com/esmiz-dream" target="_blank" rel="noopener"><img src="./images/m.png" alt="M"></a>
```
Comment (wayungi) [inline index.html:66]: - [ ] Please use the correct font for the different parts of the this  `<section class="headline">`
  - h1 font is crete round
  - h2 font is roboto , please also apply the correct color ` #FFF5E1` you can confirm it in the figma template
  - p is poppins

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @esmiz-dream , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [x] Your project is very good and thanks for the good work, Please let's fix this likely source of bugs in your code. Not that your header & nav tags are not closing correctly. Please match the tags 😄 
faulty

- [x] I have also noted that your mobile size is scrollable horizontally. This is caused by elements having a width wider than the device screen. Please fix this. The image below shows the issue 🤝 
scrollable



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (esmiz-dream) [reply index.html:67]: thank you bro

File: index.html [hunk 2]
```diff
@@ -41,33 +41,289 @@
                     <div class="menu-icon"></div>
                     <div class="menu-icon"></div>
                     <div class="menu-icon"></div>
+                </nav>
```
Comment (emmiiorji) [inline index.html:44]: [Optional] I think that you may remove the network bar part of the design, it only shows it would look on a mobile device.

image

**Review by emmiiorji — APPROVED** [review]
Hi @esmiz-dream, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — COMMENTED** [review]
Hi @esmiz-dream 👋👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses same branch for two different milestones🥲 It is a requirement that every milestone should be done on a different branch with a seperate pull request😉 kindly checkout into a new branch, open a new pull request with a descriptive header and summary🤗🤗

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**